dcp_prototype/backend/browser/.gitignore

```diff
@@ -0,0 +1,48 @@
+# Reminder:
```

Comment:
Do we need this `.gitignore`? This one looks similar to the top level one

Reply:
I guess we can remove it

Reply:
:+1: If there's significant overlap, I would vote to merge and centralize in the top level